src/cmd/tools/m3ctl/main/database/cmd.go

```diff
@@ -0,0 +1,95 @@
+package database
+
+import (
+	"flag"
+	"fmt"
+	"github.com/m3db/m3/src/cmd/tools/m3ctl/main/errors"
+	"github.com/m3db/m3/src/x/config/configflag"
+	"os"
```

Comment:
(nitpick) not sure of M3's styleguide but I'd separate stdlib imports from other imports by an empty line.